### PR TITLE
fix: Ingest main: actually initialize the connector

### PR DESCRIPTION
**Pull request by cragwolfe**
This brings back the connector initialization as was intended and got lost in a refactor.

Thanks @tomaarsen for catching this.

### CODE REVIEW

File: unstructured/ingest/main.py [hunk 1]
```diff
@@ -20,6 +20,7 @@ def __init__(self, doc_connector, doc_processor_fn, num_processes, reprocess):
 
     def initialize(self):
         """Slower initialization things: check connections, load things into memory, etc."""
+        self.doc_connector.initialize()
```
Comment (james-see) [inline unstructured/ingest/main.py:23]: Just adding this method checks the connections an everything?

Comment (james-see) [reply unstructured/ingest/main.py:23]: nevermind I see it was just pass'ing before based on the github connector PR

**Review by qued — APPROVED** [review]
LGTM!

**Comment by tomaarsen** [comment]
Looks good!

**Pull request merged**